src/EventListener/DataContainer/FilesPermissionsCallback.php

```diff
@@ -30,6 +30,10 @@ public function __construct(Security $security, RequestStack $requestStack)
 
     public function __invoke(): void
     {
+        if ('cli' === \PHP_SAPI) {
+            return;
+        }
+
         $user = $this->security->getUser();
```

Comment:
```suggestion
        if (null === ($user = $this->security->getUser())) {
            return;
        }
```

I think rather than checking for `\PHP_SAPI` I think we should just return if the user is `null`.